test/e2e/recordPlaywright.js

```diff
@@ -0,0 +1,64 @@
+const fs = require("fs");
+
+const playwright = require("@recordreplay/playwright");
+const cli = require("@recordreplay/recordings-cli");
+const _ = require("lodash");
+const { waitUntil } = require("./waitForTime");
+
+const verbose = true;
+
+async function record(browserName, example) {
+  if (verbose) {
+    console.log(`Recording Example:`, example, browserName);
+  }
+
+  const browser = await playwright[browserName].launch({
+    headless: false,
+  });
+
+  const context = await browser.newContext();
+  const page = await context.newPage();
+  await page.goto(example);
+  await waitUntil(() => page.evaluate(() => window.Example__Finished));
+  await page.close();
+  await context.close();
+  await browser.close();
+}
+
+async function upload(state, example) {
+  const list = cli.listAllRecordings();
+  const recording = _.findLast(list, rec => rec.metadata.uri == example);
```

Comment:
This works for our test suite because we put the test name in the URL, but it's probably worth flagging as something that won't work on other people's tests.

Reply:
that's fair. this code isn't trying to be generic.